boot.go

```diff
@@ -80,31 +80,31 @@ func BootHandler(w http.ResponseWriter, r *http.Request) {
 	var request_url string
 
 	// Build up the request URL
-	if mux.Vars(r)["os"] == "android" {
-		request_url = fmt.Sprintf("%sandroid/%s?%s", PEBBLE_BOOT_URL, mux.Vars(r)["path"], urlquery.Encode())
-	} else if mux.Vars(r)["os"] == "ios" {
-		request_url = fmt.Sprintf("%sios/%s?%s", PEBBLE_BOOT_URL, mux.Vars(r)["path"], urlquery.Encode())
-
+	os := mux.Vars(r)["os"]
+	if os == "android" || os == "ios" {
+		request_url = fmt.Sprintf("%s%s/%s?%s", PEBBLE_BOOT_URL, os, mux.Vars(r)["path"], urlquery.Encode())
 	} else {
 		w.Write([]byte("Invalid OS parameter"))
 		return
 	}
-
 	// Make a request to an external server then parse the request
 	req, err := http.Get(request_url)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal("Could not contact api:", err)
+	}
+	if req.StatusCode < 200 || req.StatusCode > 300 {
```

Comment:
Should we instead check if `req.StatusCode > 299` so that we are capturing anything outside of the 2xx/Success range?